pkg/monitoring/metrics/virt-controller/vmistats_collector.go

```diff
@@ -99,7 +99,7 @@ var (
 				"interface associated with the VMI in the 'address' label, and about the type of address, such as " +
 				"internal IP, in the 'type' label.",
 		},
-		[]string{"node", "namespace", "name", "network_name", "address", "type"},
+		[]string{"node", "namespace", "name", "network_name", "interface_name", "address", "type"},
```

Comment:
update network_name to vnic_name according to https://issues.redhat.com/browse/CNV-57036